policy/modules/apps/pulseaudio.te

```diff
@@ -107,20 +114,24 @@ kernel_read_kernel_sysctls(pulseaudio_t)
 
 corecmd_exec_bin(pulseaudio_t)
 
-corenet_all_recvfrom_netlabel(pulseaudio_t)
-corenet_tcp_sendrecv_generic_if(pulseaudio_t)
-corenet_udp_sendrecv_generic_if(pulseaudio_t)
-corenet_tcp_sendrecv_generic_node(pulseaudio_t)
-corenet_udp_sendrecv_generic_node(pulseaudio_t)
+tunable_policy(`pulseaudio_can_network',`
+	allow pulseaudio_t self:tcp_socket { accept listen };
```

Comment:
This should be upgraded to `create_stream_socket_perms`.

Reply:
Ready to merge